scripts/ci/serve-browser-fixtures.py

```
@@ -0,0 +1,75 @@
+
+        def log_message(self, format: str, *args: object) -> None:
+            pass
+
+    return FixtureHandler
+
+
+def main() -> None:
+    args = parse_args()
+    fixture_root = args.fixture_directory.resolve()
+    if not fixture_root.is_dir():
+        raise SystemExit(f"fixture directory does not exist: {fixture_root}")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)